### ISSUE TITLE
[MySQL] `IF EXISTS` not supported

### ISSUE THREAD

**Issue by cedx** [taozhi8833998/node-sql-parser#1053]
I'm trying to parse this MySQL statement (which is also a valid TransactSQL one):

```sql
DROP TABLE IF EXISTS event_log
```

It produces this error:

```
[SyntaxError]: Expected [A-Za-z0-9_] but " " found.
    at vs (C:\Projets\archive\mysql2sqlsrv\node_modules\node-sql-parser\index.js:1:442373)
    at Object.parse [as mariadb] (C:\Projets\archive\mysql2sqlsrv\node_modules\node-sql-parser\index.js:1:561332)
    at r.value (C:\Projets\archive\mysql2sqlsrv\node_modules\node-sql-parser\index.js:1:1114138)
    at r.value (C:\Projets\archive\mysql2sqlsrv\node_modules\node-sql-parser\index.js:1:1113818)
    at Object.<anonymous> (C:\projects\mysql2sqlsrv\mysql2sqlsrv.js:6:20)
```

When I remove the `IF EXISTS` clause, everything is OK.

**Comment by taozhi8833998** [comment]
@cedx fixed in the latest version `4.4.0`, try it.

**Comment by cedx** [comment]
Thanks @taozhi8833998 !
